### PR TITLE
Implements new design on static pages controller

### PR DESCRIPTION
**Pull request by martinemde**
I present Hammy, the new design, named after one of the featured users in the prototype: `alexand3rhammy`.

Screenshot 2024-10-08 at 8 45 23 PM

### CODE REVIEW

File: app/views/pages/faq.html.erb
Comment (martinemde) [inline app/views/pages/faq.html.erb:1]: Removed because it's very old and just links away.

File: app/views/pages/migrate.html.erb
Comment (martinemde) [inline app/views/pages/migrate.html.erb:1]: Removed because it's very very old.

**Review by indirect — APPROVED** [review]
looking good!

File: app/controllers/dashboards_controller.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ class DashboardsController < ApplicationController
   before_action :redirect_to_settings_strong_mfa_required, if: :mfa_required_weak_level_enabled?
 
   def show
+    add_breadcrumb t("breadcrumbs.dashboard"), dashboard_path
```
Comment (colby-swandale) [inline app/controllers/dashboards_controller.rb:8]: Having the breadcrumb in the action method kinda feels like it's blurring the intention. Could we take inspiration from https://github.com/weppos/breadcrumbs_on_rails and have `add_breadcrumb` as a class method?

File: app/views/components/card_component.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class CardComponent < ApplicationComponent
```
Comment (colby-swandale) [inline app/views/components/card_component.rb:3]: > This class seems somewhat complex, would it be worth putting a test together to ensure it works in various different scenarios?

Same here.

File: app/views/components/button_component.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+class ButtonComponent < ApplicationComponent
+  include Phlex::Rails::Helpers::LinkTo
+  include Phlex::Rails::Helpers::ButtonTo
+
+  attr_reader :text, :href, :type, :options, :color_css, :size_css
+
+  def initialize(text = nil, href: nil, type: :button, color: :primary, outline: false, size: :large, **options) # rubocop:disable Metrics/ParameterLists
```
Comment (colby-swandale) [inline app/views/components/button_component.rb:9]: This class seems somewhat complex, would it be worth putting a test together to ensure it works in various different scenarios?

Comment (martinemde) [reply app/views/components/button_component.rb:9]: yes, I'll follow up with that.

**Pull request merged**